lib/rucio/daemons/oauthmanager/oauthmanager.py

```diff
@@ -183,22 +182,21 @@ def run(once=False, threads=1, loop_rate=300, max_rows=100, sleep_time=300):
     sanity_check(executable='OAuthManager', hostname=socket.gethostname())
```

Comment:
this doesn't seem right. executable should be the same everywhere. Also, run_daemon actually does a sanity_check, so I would just remove this line.